src/web/app/pages-student/student-course-details-page/student-course-details-page.component.ts

```diff
@@ -99,35 +108,37 @@ export class StudentCourseDetailsPageComponent implements OnInit {
    * @param teamName: team of current student
    */
   loadTeammates(courseId: string, teamName: string): void {
+    this.teammateProfiles = [];
     this.studentService.getStudentsFromCourseAndTeam(courseId, teamName)
-        .subscribe((students: Students) => {
-          students.students.forEach((student: Student) => {
-            // filter away current user
-            if (student.email === this.student.email) {
-              return;
-            }
-
-            this.studentProfileService.getStudentProfile(student.email, courseId)
-                  .subscribe((studentProfile: StudentProfile) => {
-                    const newPhotoUrl: string =
-                      `${environment.backendUrl}/webapi/student/profilePic`
-                      + `?courseid=${courseId}&studentemail=${student.email}`;
-
-                    const newTeammateProfile: StudentProfileWithPicture = {
-                      studentProfile: {
-                        ...studentProfile,
-                        email: student.email,
-                        shortName: student.name,
-                      },
-                      photoUrl : newPhotoUrl,
-                    };
-
-                    this.teammateProfiles.push(newTeammateProfile);
-                  });
-          });
-        }, (resp: ErrorMessageOutput) => {
-          this.statusMessageService.showErrorMessage(resp.error.message);
+      .subscribe((students: Students) => {
+        students.students.forEach((student: Student) => {
+          // filter away current user
+          if (student.email === this.student.email) {
+            return;
+          }
+
+          this.studentProfileService.getStudentProfile(student.email, courseId)
+                .subscribe((studentProfile: StudentProfile) => {
+                  const newPhotoUrl: string =
+                    `${environment.backendUrl}/webapi/student/profilePic`
+                    + `?courseid=${courseId}&studentemail=${student.email}`;
+
+                  const newTeammateProfile: StudentProfileWithPicture = {
+                    studentProfile: {
+                      ...studentProfile,
+                      email: student.email,
+                      shortName: student.name,
+                    },
+                    photoUrl : newPhotoUrl,
+                  };
+
+                  this.teammateProfiles.push(newTeammateProfile);
+                });
```

Comment:
Was this indentation really necessary?

Reply:
I think that lines 121 to 136 could be positioned 2 spaces back! However, I am not the author of this method, I have just added a few lines of code in order to sort teammatesProfiles[]